test/TensorFlow/integration.swift

```diff
@@ -503,17 +480,17 @@ public func testNoInlineUser() {
     noInlineUser(x)
 }
 
+// NOTE: the output for STRICT_DA does not have strong_retain or release.
```

Comment:
@mhong Please make sure this is also OK.

Reply:
Looks good!